lib/debug/repl.cpp

```diff
@@ -548,4 +529,71 @@ void Repl::addUartCommands(const std::uint8_t bus)
   }
 }
 
+void Repl::addAccelerometerCommands(const std::uint8_t bus, const std::uint8_t device_address)
+{
+  const auto optional_i2c = io::HardwareI2c::create(logger_, bus);
+  if (!optional_i2c) {
+    logger_.log(core::LogLevel::kFatal, "Failed to create I2C instance on bus %d", bus);
+    return;
+  }
+  const auto i2c = std::move(*optional_i2c);
```

Comment:
is `std::move` definitely necessary? my CAN testing didn't end up requiring it as `shared_ptr` should not be copying when it is passed around

Reply:
no, but apparently its quicker to move instead of copying it so maybe keeping the` std::move`

Reply:
looking at it yea that makes sense